src/components/Map/Map.js

```diff
@@ -951,15 +951,9 @@ export default class Map extends React.Component<Props, State> {
 
   createMarkerFromFeature = (feature: Feature, latlng: [number, number]) => {
     const properties = feature && feature.properties;
-    if (!properties) return null;
-    if (
-      !isWheelmapFeature(feature) &&
-      !properties.accessibility &&
-      !includes(EquipmentCategoryStrings, properties.category)
-    ) {
+    if (!properties) {
```

Comment:
-1: This makes some unrateable sources visible that should not be displayed… We have some sources that are not rateable, but that have places that are reachable over the UI from elevator details screens. Concept-wise this is not beautiful, but it solved the problem of navigating between elevators in the same station without (1) cluttering the map with duplicate stations and (2) without a need to merge data.

Could this do a check that the source is rateable?

Reply:
+1 Revised my opinion, let's put this in a new PR.

Reply:
I only care about the `!properties.accessibility` line, can leave in the rest.

Reply:
But we don't have sources here, so this would make the map even more horrible.

Reply:
I'd rather change the filter to remove only the items we don't want, instead of removing too much or building weird white lists of which elements should be shown, but I'm not sure to which places/sources this applies.

Reply:
Trello ticket: https://trello.com/c/SfuLlhUu/161-dont-show-gray-places-from-unrateable-sources

How about showing all places that either have

- existing accessibility
- have no accessibility, but
  -`source.isA11yRatingAllowed`
  - are equipment

?